src/main/java/uk/gov/hmcts/reform/civil/service/pininpost/DefendantPinToPostLRspecService.java

```diff
@@ -87,7 +87,7 @@ public String validateOcmcPin(String pin, String caseReference) {
             return response.headers().get("Location").stream().findFirst()
                 .orElseThrow(() -> new IllegalArgumentException("Missing 'Location' header"));
         } else {
-            log.error("Pin does not match or expired for {}", caseReference);
+            log.error("Pin does not match or expired for {}, pin {}", caseReference, pin);
```

Comment:
Can we please remove pin from logs, as it sensitive info

Reply:
It's only logging invalid pins